python/polycap.pyx

```diff
@@ -263,9 +263,14 @@ cdef class Profile:
         z = np.asarray(z, dtype=np.double)
         z = np.atleast_1d(z)
 
-        self.profile = polycap_profile_new_from_array(z.size-1, <double*> np.PyArray_DATA(ext), <double*> np.PyArray_DATA(cap), <double*> np.PyArray_DATA(z), &error)
+        cdef polycap_profile *profile = polycap_profile_new_from_arrays(z.size-1, <double*> np.PyArray_DATA(ext), <double*> np.PyArray_DATA(cap), <double*> np.PyArray_DATA(z), &error)
```

Comment:
Waarom `z.size-1` ipv `z.size`?

Reply:
ja, da's een funky overblijfsel uit Laszlo zijn code eigenlijk :P 
Is omdat die profile->nmax in C eigenlijk de maximale index van de ext,cap,z array aangeeft, en niet gewoon het aantal elementen.
Kzou het kunnen weghalen, maar dan moet ik doorheen de hele code ook overal checken of nmax nergens out of bounds zou gaan enzo. At this point I considered it easier to just keep it.